README.md

```diff
@@ -1,254 +1,422 @@
+    separately.
+
+    In PCF version 2.0+, when the service broker creates the identity for your application
+    in Conjur, it automatically adds it to a Conjur Layer representing the `Organization`
+    and `Space` where the application is deployed. These layers may be used for control secret
+    access at the org or space level, rather than the application host itself.
+
+    ##### Space-scoped Identity
+
+    Space-scoped identities are enabled by configuring the service broker with
+    `ENABLE_SPACE_IDENTITY` set to `true`. This means that when a service instance is created
+    in a space, the service broker creates a Conjur Host for that space. When an application
+    is bound to the service, the service broker gives it the credentials of the space identity,
+    rather than create a new host identity for the application.
+
+    The advantage to this is the bind operation only requires access to a Conjur follower and
+    not the Conjur master. This promotes high-availability and scalability of app binding and secret
+    retrieval.
+
+    ##### Application-scoped Identity
+
+    When space identities are not enabled, the service broker creates a new Conjur host identity
+    for each application bound to the service. This requires that the service broker is able to
+    communicate with the Conjur master for each bind request.
+
+    The advantage to this is finer-grained access control and audit logs in Conjur.
+
+3. **Permit the Application to Access Secrets in Conjur**
+
+    PCF applications can be granted access to secrets using either the Org and Space layers,
+    or with the application host identity.
+
+    #### Privilege Org and Space Layers
+
+    Applications can be granted access to secrets by privileging the Org or Space layers
+    to read secrets using Conjur policy.
+
+    The layer Ids use the Org and Space GUID identifiers, which may be obtained
+    using the Cloud Foundry CLI:
+    ```sh-session
+    $ cf org --guid <org-name>
+    6b40649e-331b-424d-afa0-6d569f016f51
+
+    $ cf space --guid <space-name>
+    72a928f6-bf7c-4732-a195-896f67bd1133
+    ```
+
+    For example, the policy to privilege a space to access a secret is:
+    ```yaml
+    - !permit
+      resource: my-secret-id
+      role: !layer cf/prod/6b40649e-331b-424d-afa0-6d569f016f51/72a928f6-bf7c-4732-a195-896f67bd1133
+      privileges: [ read, execute ]
+    ```
+
+    #### Privilege Application Host Identity
 
-### Run Your Application
+    > **NOTE:** Application Host privileging is not available when using Space Host Identies.
 
-Now that your application is privileged to access the secrets it needs in Conjur,
-start or restage the app so that the Conjur Buildpack can inject the secret values
-into the running application's environment.
+    After your application has been pushed to PCF, you can use its host identity in
+    Conjur policy to grant it access to secrets.
```

Comment:
There should be a note here that the app can't be started and be able to retrieve secrets until this step has been completed